src/tests/components/Card.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
You need some more tests here, currently you are just testing the title and body, we also want to test the image and links etc